Realm/RLMObject.h

```diff
@@ -307,7 +309,7 @@
  @return    An object of the subclass type or nil if an object with the given primary key does not exist.
  @see       -primaryKey
  */
-+ (instancetype)objectForPrimaryKey:(id)primaryKey;
++ (nullable instancetype)objectForPrimaryKey:(id)primaryKey;
```

Comment:
Probably not useful, no, but forbidding it when it'd technically work seems weird.